File: game/src/main/scala/falcun/net/api/gui/menu/FalcunPage.scala
package falcun.net.api.gui.menu

import falcun.net.api.gui.components.Component

import java.util
import java.util.List


trait FalcunPage {
  def getComponents: util.List[Component]

  def getBackgroundWidth = 1035

  def getBackgroundHeight = 485
}

File: client/projects/Forge/src/main/java/falcun/net/api/gui/menu/FalcunPage.java
package falcun.net.api.gui.menu;

import falcun.net.api.gui.components.Component;

import java.util.List;

public interface FalcunPage {
	List<Component> getComponents();
	default int getBackgroundWidth(){
		return 1035;
	}
	default int getBackgroundHeight(){
		return 485;
	}
}
